### PR TITLE
[WFCORE-4973] Increasing LongOutputTestCase timeout from 10 to 20 seconds

### PR DESCRIPTION
**Pull request by ropalka**
https://issues.redhat.com/browse/WFCORE-4973

### CODE REVIEW

File: testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/LongOutputTestCase.java [hunk 1]
```diff
@@ -389,7 +389,7 @@ public void testEnableOutputPagingViaXml() throws Exception {
     private void testDisabledOutputPaging() throws Exception {
         consoleWriter.println("/subsystem=elytron:read-resource-description(recursive=true)");
         Assert.assertFalse(consoleWriter.checkError());
-        String window = queue.poll(10, TimeUnit.SECONDS);
+        String window = queue.poll(100, TimeUnit.SECONDS);
```
Comment (jmesnil) [inline testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/LongOutputTestCase.java:392]: 100 seconds is 1 order of magnitude more. It seems a lot.

Would doubling to 20s be enough?
Or could we instead use org.jboss.as.test.shared.TimeoutUtil#adjust to adjust the time of this test?

Comment (ropalka) [reply testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/LongOutputTestCase.java:392]: This problem pops up if I'm running whole WildFly test suite only (~ 2.5 hours).
If I run the test standalone, it always passes for me. 
You are right @jmesnil 100 seconds is much more higher but
test usually don't spend so much time in it. It is just safer upper bound.
If you still prefer 20 seconds instead, please let me know - I will change it and will test it locally.

Comment (jmesnil) [reply testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/LongOutputTestCase.java:392]: @ropalka Let's double it and use org.jboss.as.test.shared.TimeoutUtil#adjust to adjust it automatically when it is run in our CI, thanks

**Comment by ropalka** [comment]
I doubled the timeout value + used TimeoutUtil.adjust() method as you suggested @jmesnil

**Comment by ropalka** [comment]
Fixed @darranl - thanks for spotting it out.

**Comment by darranl** [comment]
CI errors appear unrelated, adding to ready to merge batch.

**Pull request merged**